test/integration/user/userRemove.test.ts

```diff
@@ -14,6 +15,8 @@ export default () =>
       user,
     })
 
+    await UserController.remove({ user: await UserController.read({ user: userMockUnknown }) })
```

Comment:
UserController.remove should not be there.
in our test we should run delete statement directly here